go/cli/mcap/cmd/doctor.go

```diff
@@ -182,13 +183,23 @@ func (doctor *mcapDoctor) examineChunk(chunk *mcap.Chunk) {
 				doctor.error("Got a Message record for channel: %d before a channel info.", message.ChannelID)
 			}
 
+			if strictMessageOrder && message.LogTime < doctor.maxLogTime {
+				doctor.error("Message.log_time %d on %s is less than the oldest log time %d",
+					message.LogTime, channel.Topic, doctor.maxLogTime)
+			}
+
 			if message.LogTime < minLogTime {
 				minLogTime = message.LogTime
 			}
 
 			if message.LogTime > maxLogTime {
 				maxLogTime = message.LogTime
```

Comment:
do we still need this local variable `maxLogTime`?

Reply:
I think maybe not? Its not clear to me why these were not updating the `doctor` instance directly